src/commands/init.js

```diff
@@ -11,19 +11,37 @@ module.exports = class extends Command {
 			{ label: 'dirigeants/klasa', description: 'From Github repository' }
 		], { placeHolder: 'Select source:' });
 
-		if (!repo) throw undefined;
-		const terminal = window.createTerminal('Klasa');
+		const { label: lang } = await window.showQuickPick([
+			{ label: 'JavaScript', description: 'Use JavaScript for the bot' },
+			{ label: 'TypeScript', description: 'Use TypeScript for the bot' }
+		], { placeHolder: 'Select language:' });
+
+		const { label: pkgManager } = await window.showQuickPick([
+			{ label: 'npm', description: 'Use NPM to install packages' },
+			{ label: 'yarn', description: 'Use yarn to install packages' }
+		], { placeHolder: 'Select package manager:' });
 
-		const editor = await this.createFile(resolve(mainDir, '.gitignore'), 'ignore file');
-		await editor.document.save();
-		const editor2 = await this.createFile(resolve(baseDir, 'index.js'), 'entry file');
-		await editor2.document.save();
-		await this.createFile(resolve(baseDir, 'config.js'), 'config file');
+		if (!repo || !lang || !pkgManager) throw undefined;
+
+		const terminal = window.createTerminal('Klasa');
+		if (lang === 'JavaScript') {
+			const editor = await this.createFile(resolve(mainDir, '.gitignore'), 'ignore file');
+			await editor.document.save();
+			const editor2 = await this.createFile(resolve(baseDir, 'index.js'), 'entry file');
+			await editor2.document.save();
+			await this.createFile(resolve(baseDir, 'config.js'), 'config file');
+		} else {
+			const editor = await this.createFile(resolve(mainDir, '.gitignore'), 'ignore file');
```

Comment:
You can refactor this by moving lines 28-29 and 34-35 in the line 27 (before `if (lang === 'JavaScript') {`), as both are identical.